tests/index.test.ts

```diff
@@ -411,3 +411,59 @@ describe('retries', () => {
     expect(count).toEqual(3);
   });
 });
+
+describe('Debug', () => {
+  const env = process.env;
+  const spy = jest.spyOn(console, 'log');
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = { ...env };
+    process.env['DEBUG'] = 'true';
+  });
+
+  afterEach(() => {
+    process.env = env;
+  });
+
+  test('body request object with Authorization header', function () {
```

Comment:
can you also add a test to ensure the inputs aren't mutated?

Reply:
Updated